src/azure-cli-core/azure/cli/core/auth/persistence.py

```diff
@@ -67,3 +67,5 @@ def load(self):
             return json.loads(self._persistence.load())
         except PersistenceNotFound:
             return []
+        except Exception:
+            return []
```

Comment:
The raised error is `azure.cli.core.vendored_sdks.dummy_msal_extensions.persistence.PersistenceNotFound`.
If we use `dummy_msal_extensions` globally, the imported Exception type will change when use `from` import statement.
```python
sys.modules['msal_extensions'] = dummy_msal_extensions

import msal_extensions
print(type(msal_extensions.persistence.PersistenceNotFound()))
from msal_extensions.persistence import PersistenceNotFound
print(type(PersistenceNotFound()))
print(type(msal_extensions.persistence.PersistenceNotFound()))
```
```
>>> <class 'azure.cli.core.vendored_sdks.dummy_msal_extensions.persistence.PersistenceNotFound'>
>>> <class 'msal_extensions.persistence.PersistenceNotFound'>
>>> <class 'msal_extensions.persistence.PersistenceNotFound'>
```